components/Icon/Icon.css

```diff
@@ -12,18 +12,26 @@
   line-height: inherit;
 }
 
-/* Types */
+.pactH {
+  height: 1.1em;
+  width: calc(1.1em * 2.272727273);
+}
+
```

Comment:
Is this 2.2727272727hisgdh782t568922th389 specific for here or is it a scale var we might reuse?